commands/command.md

```diff
@@ -168,11 +202,112 @@ like `MSET` where the format is `MSET _key1_ _val1_ [key2] [val2] [key3] [val3].
 In the case of `MSET`, keys are every other position so the step value is 2.  Compare
 with `MGET` above where the step value is just 1.
 
+## acl categories
+
+Available starting from Redis 6.0
+
+For more information please check the [acl page][tr].
+[tr]: /topics/acl
+
+## additional information
+
+Available starting from Redis 7.0
+
+The last element of each element in `COMMAND`s reply is a a map with the following fields:
```

Comment:
i think that "last" is not good, we need to specify the exact indexes (also for other fields)